MoltenVK/MoltenVK/Commands/MVKCmdRenderPass.mm

```diff
@@ -61,13 +85,23 @@
 template <size_t N_CV, size_t N_A>
 void MVKCmdBeginRenderPass<N_CV, N_A>::encode(MVKCommandEncoder* cmdEncoder) {
 //	MVKLogDebug("Encoding vkCmdBeginRenderPass(). Elapsed time: %.6f ms.", mvkGetElapsedMilliseconds());
+
+	// Convert the sample position array of arrays to an array of array-references,
+	// so that it can be passed to the command encoder.
+	size_t spSPCnt = _subpassSamplePositions.size();
+	MVKArrayRef<MTLSamplePosition> spSPRefs[spSPCnt];
+	for (uint32_t spSPIdx = 0; spSPIdx < spSPCnt; spSPIdx++) {
+		spSPRefs[spSPIdx] = _subpassSamplePositions[spSPIdx].contents();
+	}
+	
 	cmdEncoder->beginRenderpass(this,
 								_contents,
 								_renderPass,
 								_framebuffer,
 								_renderArea,
 								_clearValues.contents(),
-								_attachments.contents());
+								_attachments.contents(),
+								MVKArrayRef<MVKArrayRef<MTLSamplePosition>>(spSPRefs, spSPCnt));
```

Comment:
There should really be an `mvkMakeArrayRef()` function. Then we wouldn't need to repeat the template arguments.

Reply:
Ah. Good catch and suggestion. Fix applied...but it turned out the `MVKArrayRef` constructor itself was sufficient to take care of this.